apps/condo/domains/notification/schema/_internalSendNotificationNewMobileAppVersionService.js

```diff
@@ -0,0 +1,140 @@
+
+const _internalSendNotificationNewMobileAppVersionService = new GQLCustomSchema('_internalSendNotificationNewMobileAppVersionService', {
+    types: [
+        {
+            access: true,
+            type: `enum App { ${APP_RESIDENT_KEY}, ${APP_MASTER_KEY} }`,
+        },
+        {
+            access: true,
+            type: `enum Platform { ${DEVICE_PLATFORM_ANDROID}, ${DEVICE_PLATFORM_IOS} }`,
+        },
+        {
+            access: true,
+            type: 'input _internalSendNotificationNewMobileAppVersionInput { dv: Int!, sender: JSON! platform: Platform!, app: App!, buildVersion: String!, title: String, body: String, organizationIds: [ID!] }',
+        },
+        {
+            access: true,
+            type: 'type _internalSendNotificationNewMobileAppVersionOutput { messageBatchId: ID! }',
+        },
+    ],
+    
+    mutations: [
+        {
+            access: access.can_internalSendNotificationNewMobileAppVersion,
+            schema: '_internalSendNotificationNewMobileAppVersion(data: _internalSendNotificationNewMobileAppVersionInput!): _internalSendNotificationNewMobileAppVersionOutput',
+            resolver: async (parent, args, context, info, extra = {}) => {
+
+                const { data: { dv, sender, platform, app, buildVersion, title, body, organizationIds } } = args
+
+                const { keystone } = await getSchemaCtx('MessageBatch')
+                const adminContext = await keystone.createContext({ skipAccessControl: true })
+
+                const userType = app === RESIDENT ? RESIDENT : STAFF
+                let remoteClientKeys
+                //The case when a push is sent to all suitable devices, regardless of organization
+                if (isEmpty(organizationIds)) {
+                    remoteClientKeys = await loadListByChunks({
+                        context: adminContext,
+                        list: RemoteClient,
+                        where: {
+                            devicePlatform: platform,
+                            deletedAt: null,
+                            owner: {
+                                type: userType,
+                            },
+                        },
+                        chunkSize: 50,
+                        chunkProcessor: (/** @type {RemoteClient[]} */ chunk) => {
+                            const rcWithLowerBuild = []
+
+                            if (buildVersion) {
+                                for (const rc of chunk) {
+                                    if (get(rc, 'meta.Build', buildVersion) < buildVersion) rcWithLowerBuild.push(`rc:${rc.id}`)
+                                }
+                            }
+
+                            return rcWithLowerBuild
+                        },
+                    })
+                }
+
+                //The case when a push is sent to suitable devices of a specific or several organizations
+                if (!isEmpty(organizationIds)) {
```

Comment:
Why not just `else`?
```suggestion
                } else { //The case when a push is sent to suitable devices of a specific or several organizations
```